.codecov.yml

```diff
@@ -0,0 +1,18 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: 50..100
```

Comment:
Any reason for these values in particular? Can you add a comment?

Reply:
Yeah, in fact I'll change that to 50..75 meaning that anything less than 50% is considered bad and is marked as red, and the more we approach 75% the greener the graphs will get. I don't think we should only make 100% green since that implies that only 100% test coverage is "healthy".

Test coverage can be a misleading metric, not saying anything about the quality of the tests and sometimes even encouraging unrealistic and useless tests. Treating 75% as a goal is a good compromise that encourages test writing but staying in the realm of encouragement instead of penalization.